===================================================================
RCS file: /cvs/xml/oox/source/core/binarycodec.cxx,v
retrieving revision 1.1.2.2
diff -u -u -r1.1.2.2 binarycodec.cxx
--- oox/source/core/binarycodec.cxx	28 Mar 2007 14:27:16 -0000	1.1.2.2
+++ oox/source/core/binarycodec.cxx	25 Jan 2008 14:34:58 -0000
@@ -35,6 +35,7 @@
 
 #include "oox/core/binarycodec.hxx"
 #include <algorithm>
+#include <string.h>
 #include <osl/diagnose.h>
 
 namespace oox {
Index: binaryinputstream.cxx
===================================================================
RCS file: /cvs/xml/oox/source/helper/binaryinputstream.cxx,v
retrieving revision 1.1.2.2
diff -u -u -r1.1.2.2 binaryinputstream.cxx
--- oox/source/helper/binaryinputstream.cxx	7 Dec 2007 10:22:00 -0000	1.1.2.2
+++ oox/source/helper/binaryinputstream.cxx	25 Jan 2008 14:37:22 -0000
@@ -36,6 +36,7 @@
 #include "oox/helper/binaryinputstream.hxx"
 #include <com/sun/star/io/XInputStream.hpp>
 #include <osl/diagnose.h>
+#include <string.h>
 
 using ::com::sun::star::uno::Reference;
 using ::com::sun::star::uno::Sequence;
Index: binaryoutputstream.cxx
===================================================================
RCS file: /cvs/xml/oox/source/helper/binaryoutputstream.cxx,v
retrieving revision 1.1.2.2
diff -u -u -r1.1.2.2 binaryoutputstream.cxx
--- oox/source/helper/binaryoutputstream.cxx	7 Dec 2007 10:22:00 -0000	1.1.2.2
+++ oox/source/helper/binaryoutputstream.cxx	25 Jan 2008 14:38:14 -0000
@@ -37,6 +37,7 @@
 #include <osl/diagnose.h>
 #include <com/sun/star/io/XOutputStream.hpp>
 #include "oox/helper/binaryinputstream.hxx"
+#include <string.h>
 
 using ::com::sun::star::uno::Reference;
 using ::com::sun::star::uno::Sequence;
Index: recordinputstream.cxx
===================================================================
RCS file: /cvs/xml/oox/source/helper/recordinputstream.cxx,v
retrieving revision 1.1.2.1
diff -u -u -r1.1.2.1 recordinputstream.cxx
--- oox/source/helper/recordinputstream.cxx	5 Dec 2007 14:23:59 -0000	1.1.2.1
+++ oox/source/helper/recordinputstream.cxx	25 Jan 2008 14:40:25 -0000
@@ -35,6 +35,7 @@
 
 #include "oox/helper/recordinputstream.hxx"
 #include <vector>
+#include <string.h>
 
 using ::rtl::OUString;
 
Index: commonbehaviorcontext.cxx
===================================================================
RCS file: /cvs/xml/oox/source/ppt/commonbehaviorcontext.cxx,v
retrieving revision 1.1.2.6
diff -u -u -r1.1.2.6 commonbehaviorcontext.cxx
--- oox/source/ppt/commonbehaviorcontext.cxx	16 Nov 2007 00:08:11 -0000	1.1.2.6
+++ oox/source/ppt/commonbehaviorcontext.cxx	25 Jan 2008 14:42:37 -0000
@@ -52,6 +52,7 @@
 #include "pptfilterhelpers.hxx"
 #include "tokens.hxx"
 
+#include <string.h>
 
 using ::rtl::OUString;
 using ::rtl::OUStringBuffer;
Index: biffcodec.cxx
===================================================================
RCS file: /cvs/xml/oox/source/xls/biffcodec.cxx,v
retrieving revision 1.1.2.5
diff -u -u -r1.1.2.5 biffcodec.cxx
--- oox/source/xls/biffcodec.cxx	5 Dec 2007 14:28:22 -0000	1.1.2.5
+++ oox/source/xls/biffcodec.cxx	25 Jan 2008 14:44:28 -0000
@@ -35,6 +35,7 @@
 
 #include "oox/xls/biffcodec.hxx"
 #include <osl/thread.h>
+#include <string.h>
 
 using ::rtl::OString;
 using ::rtl::OUString;
Index: inc/oox/helper/helper.hxx
===================================================================
RCS file: /cvs/xml/oox/inc/oox/helper/helper.hxx,v
retrieving revision 1.1.2.2
diff -u -u -r1.1.2.2 helper.hxx
--- oox/inc/oox/helper/helper.hxx	7 Dec 2007 10:21:37 -0000	1.1.2.2
+++ oox/inc/oox/helper/helper.hxx	25 Jan 2008 14:46:05 -0000
@@ -40,6 +40,7 @@
 #include <osl/endian.h>
 #include <rtl/string.hxx>
 #include <rtl/ustring.hxx>
+#include <string.h>
 
 namespace oox {
 
Index: QName.cxx
===================================================================
RCS file: /cvs/sw/writerfilter/source/odiapi/qname/Attic/QName.cxx,v
retrieving revision 1.2
diff -u -u -r1.2 QName.cxx
--- writerfilter/source/odiapi/qname/QName.cxx	1 Nov 2006 09:14:35 -0000	1.2
+++ writerfilter/source/odiapi/qname/QName.cxx	25 Jan 2008 22:18:26 -0000
@@ -60,6 +60,7 @@
 #endif
 
 #include <stdio.h>
+#include <string.h>
 
 namespace writerfilter
 {
Index: OOXMLBinaryObjectReference.cxx
===================================================================
RCS file: /cvs/sw/writerfilter/source/ooxml/OOXMLBinaryObjectReference.cxx,v
retrieving revision 1.1.6.3
diff -u -u -r1.1.6.3 OOXMLBinaryObjectReference.cxx
--- writerfilter/source/ooxml/OOXMLBinaryObjectReference.cxx	14 Nov 2007 12:58:58 -0000	1.1.6.3
+++ writerfilter/source/ooxml/OOXMLBinaryObjectReference.cxx	25 Jan 2008 22:22:44 -0000
@@ -33,6 +33,7 @@
  *
  ************************************************************************/
 #include "OOXMLBinaryObjectReference.hxx"
+#include <string.h>
 
 namespace writerfilter {
 namespace ooxml
--- writerfilter/source/ooxml/makefile.mk-old	2008-01-25 23:43:25.000000000 +0100
+++ writerfilter/source/ooxml/makefile.mk	2008-01-25 23:43:40.000000000 +0100
@@ -208,7 +208,7 @@
 	$(XSLTPROC) $(OOXMLRESOURCEIDSXSL) $(MODELPROCESSED) > $@
 
 $(GPERFFASTTOKENHXX): $(OOXMLGPERFFASTTOKENXSL) $(MODELPROCESSED)
-	$(XSLTPROC) $(OOXMLGPERFFASTTOKENXSL) $(MODELPROCESSED) | tr -d '\r' | gperf -t -E -S1 -c -G -LC++ > $@
+	$(XSLTPROC) $(OOXMLGPERFFASTTOKENXSL) $(MODELPROCESSED) | tr -d '\r' | gperf -I -t -E -S1 -c -G -LC++ > $@
 
 $(SLOFASTNAMESPACES): $(OOXMLFASTRESOURCESCXX) $(OOXMLGENHEADERS)
 
--- writerfilter/source/dmapper/DomainMapper_Impl.hxx-old	2008-01-25 23:59:14.000000000 +0100
+++ writerfilter/source/dmapper/DomainMapper_Impl.hxx	2008-01-26 00:00:05.000000000 +0100
@@ -92,6 +92,8 @@
 #pragma GCC visibility pop
 #endif
 
+#include <string.h>
+
 namespace com{ namespace sun{ namespace star{
         namespace lang{
             class XMultiServiceFactory;
--- writerfilter/source/dmapper/GraphicImport.cxx-old	2008-01-26 00:01:31.000000000 +0100
+++ writerfilter/source/dmapper/GraphicImport.cxx	2008-01-26 00:01:43.000000000 +0100
@@ -107,6 +107,7 @@
 
 #include <iostream>
 #include <resourcemodel/QNameToString.hxx>
+#include <string.h>
 
 namespace writerfilter {
 namespace dmapper
--- writerfilter/unocomponent/debugservices/rtfsl/RTFSLParser.cxx	19 Jun 2007 05:39:12 -0000	1.5
+++ writerfilter/unocomponent/debugservices/rtfsl/RTFSLParser.cxx	25 Jan 2008 23:04:47 -0000
@@ -39,6 +39,7 @@
 
 #include "RTFSLParser.hxx"
 #include <stdio.h>
+#include <string.h>
 #include <wchar.h>
 #include <rtftok/RTFScanner.hxx>
 #include <rtftok/RTFScannerHandler.hxx>
--- writerfilter/unocomponent/debugservices/rtftok/ScannerTestService.cxx-old	2008-01-26 00:06:10.000000000 +0100
+++ writerfilter/unocomponent/debugservices/rtftok/ScannerTestService.cxx	2008-01-26 00:06:26.000000000 +0100
@@ -39,6 +39,7 @@
 
 #include "ScannerTestService.hxx"
 #include <stdio.h>
+#include <string.h>
 #include <wchar.h>
 #include <rtftok/RTFScanner.hxx>
 #include <rtftok/RTFScannerHandler.hxx>
--- writerfilter/unocomponent/debugservices/rtftok/XMLScanner.cxx-old	2008-01-26 00:07:50.000000000 +0100
+++ writerfilter/unocomponent/debugservices/rtftok/XMLScanner.cxx	2008-01-26 00:08:02.000000000 +0100
@@ -39,6 +39,7 @@
 
 #include "XMLScanner.hxx"
 #include <stdio.h>
+#include <string.h>
 #include <wchar.h>
 #include <rtftok/RTFScanner.hxx>
 #include <rtftok/RTFScannerHandler.hxx>

